src/components/feature_activation/SignalBitRow.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import featureActivation from '../../utils/featureActivation';
+
+
+class SignalBitRow extends React.Component {
+  render() {
+    const { bit, signal, feature, feature_state } = this.props.signalBit
```

Comment:
We should be more descriptive, `signalBit` does not seem like a good name for an object with the properties bit, signal, feature, feature_state.
Maybe `featureData` for a `FeatureDataRow` component would be more indicative of the what we will use this component for, what do you think?

Reply:
Sure! Done in https://github.com/HathorNetwork/hathor-explorer/pull/244/commits/c78ca0f1ff3118c0ac115e302d0f8dc15d8c651e